_data/simple-icons.json

```
@@ -2280,6 +2280,11 @@
             "hex": "6383A8",
             "source": "https://vk.com/about"
         },
+        {
+            "title": "VLC media player",
+            "hex": "FF8800",
+            "source": "http://git.videolan.org/?p=vlc.git;a=blob_plain;f=extras/package/macosx/asset_sources/"
```

For me this link doesn't work but this one does?

```suggestion
            "source": "http://git.videolan.org/?p=vlc.git;a=tree;f=extras/package/macosx/asset_sources"
```